tcs-api/src/main/java/com/transformuk/hee/tis/tcs/api/dto/PostEsrEventDto.java

```diff
@@ -0,0 +1,21 @@
+package com.transformuk.hee.tis.tcs.api.dto;
+
+import com.transformuk.hee.tis.tcs.api.enumeration.PostEsrEventStatus;
+import java.io.Serializable;
+import java.util.Date;
+import lombok.Data;
+
+/**
+ * This DTO is used for ESR Reconcilation Event for Post.
+ */
+@Data
+public class PostEsrEventDto implements Serializable {
+  private static final long serialVersionUID = 1L;
+
+  private Date reconciledAt;
```

Comment:
Let's stick to the same names.  Also, I think using `LocalDateTime` is more appropriate here; although display might only require the date, it will have a time too.  I'm not so sure the "reconciled" date is something we will have where as we will have a "Source Event" datetime.